base/siteorigin-widget.class.php

```diff
@@ -727,23 +727,25 @@ private function save_css( $instance ){
 
 		$css = $this->get_instance_css($instance);
 
-		if( !empty($css) ) {
-
+		if ( ! empty( $css ) ) {
 			if ( WP_Filesystem() ) {
 				global $wp_filesystem;
 				$upload_dir = wp_upload_dir();
 
 				if ( ! $wp_filesystem->is_dir( $upload_dir['basedir'] . '/siteorigin-widgets/' ) ) {
-					$wp_filesystem->mkdir( $upload_dir['basedir'] . '/siteorigin-widgets/' );
+					$add_cache = $wp_filesystem->mkdir( $upload_dir['basedir'] . '/siteorigin-widgets/' );
```

Comment:
`$wp_filesystem->mkdir` returns `false` if it fails, so this should be `$add_cache = ! $wp_filesystem->mkdir(...` as we only want to use the cache if we fail to write the file, correct? Alternatively, if you want to avoid the negation, you should rename `$add_cache`.